doorknob/test_detect.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+import boto3
+import re
+
+from unittest import mock
+from freezegun import freeze_time
+from detect import VideoDetect
+
+from moto import mock_sns, mock_sqs
```

Comment:
The imports in these lines are also causing ```ModuleNotFound``` errors. Would you please make sure to freeze your requirements on ```requirements/local.txt``` and ```requirements/production.txt``` files if you installed new packages?